### PR TITLE
Allow missing timestamps or same consecutive timestamps

### LINKED ISSUE #6: Consecutive points with same timestamp → div by zero
Consider a GPX file with these four points:

    <trkpt lat="46.585705" lon="6.604863"><ele>603.2</ele><time>2016-02-15T06:34:26Z</time></trkpt>
    <trkpt lat="46.585689" lon="6.604812"><ele>603.2</ele><time>2016-02-15T06:34:27Z</time></trkpt>
    <trkpt lat="46.585658" lon="6.604792"><ele>603.2</ele><time>2016-02-15T06:34:27Z</time></trkpt>
    <trkpt lat="46.585623" lon="6.604806"><ele>603.2</ele><time>2016-02-15T06:34:28Z</time></trkpt>

Lines 2 and 3 have the same timestamp. Different positions though.

PHP-GPX-Ingest currently fails on such input with:

    Warning: Division by zero in GPXIngest->calculateAcceleration() (line 597 of … GPXIngest.class.php).

Before possibly fixing GPXIngest code, I'd like to know: Do you know if it is valid according to the GPX spec to have 2 points with same timestamp?

### PR DESCRIPTION
**Pull request by fbonzon**
Fixes #6 

+ Remove trailing spaces (sorry, done automatically by my text editor)

Avoids division by zero errors.

### CODE REVIEW

File: GPXIngest.class.php [hunk 1]
```diff
@@ -546,12 +546,24 @@ public function ingest(){
 			$this->journey->stats->minSpeed = min($this->lowspeeds);
 			$this->journey->stats->modalSpeed = array_search(max($modesearch),$modesearch);
 			$this->journey->stats->avgspeed = round(array_sum($this->journeyspeeds) / $this->journey->stats->trackpoints,2);
-			$this->journey->stats->maxacceleration = max($this->accels);
-			$this->journey->stats->maxdeceleration = max($this->decels);
-			$this->journey->stats->minacceleration = min($this->accels);
-			$this->journey->stats->mindeceleration = min($this->decels);
-			$this->journey->stats->avgacceleration = round(array_sum($this->accels)/count($this->accels),2);
-			$this->journey->stats->avgdeceleration = round(array_sum($this->decels)/count($this->accels),2);
```
Comment (fbonzon) [inline GPXIngest.class.php]: Changed accels to decels here too, which I believe was a bug.

**Comment by fbonzon** [comment]
Friendly ping.

**Comment by bentasker** [comment]
Sorry, must've missed the original notification!

Looks good to me, merging to fix #6

**Comment by bentasker** [comment]
Thanks!

**Pull request merged**